manageiq-operator/deploy/crds/manageiq.org_manageiqs_crd.yaml

```diff
@@ -0,0 +1,161 @@
+                objects'
+              type: string
+            applicationAdminPassword:
+              description: admin user initial password
+              type: string
+            applicationDomain:
+              description: Domain name for the external route Used for external authentication
+                configuration
+              type: string
+            databaseRegion:
+              description: Application region number
+              type: string
+            databaseSecret:
+              type: string
+            databaseVolumeCapacity:
+              description: Containerized database volume size
+              type: string
+            encryptionKey:
+              type: string
+            httpdCpuRequest:
+              type: string
+            httpdImageName:
+              type: string
+            httpdImageTag:
+              type: string
+            httpdMemoryLimit:
+              type: string
+            httpdMemoryRequest:
+              type: string
+            kafkaSecret:
+              description: kafka deployment information
+              type: string
+            kafkaVolumeCapacity:
+              type: string
+            memcachedCpuRequest:
+              description: memcachedpod deployment information
+              type: string
+            memcachedImageName:
+              type: string
+            memcachedImageTag:
+              type: string
+            memcachedMaxConnection:
+              type: string
+            memcachedMaxMemory:
+              type: string
+            memcachedMemoryLimit:
+              type: string
+            memcachedMemoryRequest:
+              type: string
+            memcachedSlabPageSize:
+              type: string
+            orchestratorCpuRequest:
+              description: ' orchestrator deployment information'
+              type: string
+            orchestratorImageName:
+              type: string
+            orchestratorImageNamespace:
+              type: string
+            orchestratorImageTag:
+              type: string
+            orchestratorMemoryLimit:
+              type: string
+            orchestratorMemoryRequest:
+              type: string
+            postgresqlCpuRequest:
+              description: postgres database pod deployment information
+              type: string
+            postgresqlImageName:
+              type: string
+            postgresqlImageTag:
+              type: string
+            postgresqlMaxConnections:
+              type: string
+            postgresqlMemoryLimit:
+              type: string
+            postgresqlMemoryRequest:
+              type: string
+            postgresqlSharedBuffers:
+              type: string
+            zookeeperVolumeCapacity:
+              type: string
+          required:
+          - appName
+          - applicationAdminPassword
+          - applicationDomain
+          - databaseRegion
+          - databaseVolumeCapacity
+          - encryptionKey
```

Comment:
This is not required. It will be generated if not passed.

In fact, this should just be removed. I'll handle that in a follow-up.